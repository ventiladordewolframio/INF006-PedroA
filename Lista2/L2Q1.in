5 8 -1 4 6 9 11 15
5 6 7 6 5 4