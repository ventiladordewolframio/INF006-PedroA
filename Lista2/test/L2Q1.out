0 1 1 2 2 2 3 4 max 15 alt 4 pred 11
0 1 2 3 2 1 max 7 alt 2 pred 6